libs/native-federation/src/utils/i18n.ts

```diff
@@ -76,7 +76,9 @@ export async function translateFederationArtefacts(
 
   const sourceLocalePath = path.join(outputPath, 'browser', sourceLocale);
 
-  const cmd = `node node_modules/.bin/localize-translate -r ${sourceLocalePath} -s "${sourcePattern}" -t ${translationFiles} -o ${translationOutPath} --target-locales ${targetLocales} -l ${sourceLocale}`;
```

Comment:
here executing `execSync("node node_modules/.bin/localize-translate ... ")` where "localize-translate" contains
```
@ECHO off
GOTO start
:find_dp0
SET dp0=%~dp0
EXIT /b
:start
SETLOCAL
CALL :find_dp0

IF EXIST "%dp0%\node.exe" (
  SET "_prog=%dp0%\node.exe"
) ELSE (
  SET "_prog=node"
  SET PATHEXT=%PATHEXT:;.JS;=;%
)

endLocal & goto #_undefined_# 2>NUL || title %COMSPEC% & "%_prog%"  "%dp0%\..\@angular\localize\tools\bundles\src\translate\cli.js" %*

```

does not make much sense, no? we execute it a bit node wrapped, something like `node node translate -args` i guess.